utils.py

```diff
@@ -86,27 +86,27 @@ def train(model, dataloaders, optimizer, criterion, device, num_epochs=10, callb
                 break
 
 
-def test(model, dataloaders, criterion, device):
+def test(model, dataloader, criterion, params):
     """Test the model using the testloader."""
     model.eval()
     test_loss = 0
     test_correct = 0
     test_inputs = 0
 
-    bar = tqdm(enumerate(dataloaders['Test']), total=len(dataloaders['Test']), desc='Test: ')
-
-    for batch, (inputs, labels) in bar:
-        inputs = inputs.to(device)
-        labels = labels.to(device)
-
-        outputs = model(inputs)
-        loss = criterion(outputs, labels)
-
-        probabilities_task = F.softmax(outputs, dim=1)
-        _, predicted_task = torch.max(probabilities_task, 1)
-        test_loss += loss.sum().item()
-        test_correct += (predicted_task == labels).sum().item()
-        test_inputs += len(labels)
-        bar.set_postfix_str(f'Loss Test: {test_loss / test_inputs:.4f}, '
-                            f'Acc Test: {test_correct / test_inputs:.4f}')
+    bar = tqdm(enumerate(dataloader['Test']), total=len(dataloader['Test']), desc='Test: ')
```

Comment:
As the code is right now, dataloader (testloader) is not a dictionary, so it will raise an error.

Reply:
Done didn't see that thank you